bindings/python/tests/test_core.py

```diff
@@ -19,39 +19,43 @@
 
 def test_blocking():
     op = opendal.Operator("memory")
-    op.write("test", b"Hello, World!")
-    bs = op.read("test")
+    op.create_dir("test/")
+    op.write("test/test", b"Hello, World!")
```

Comment:
By the way, there's no need to create the directory before writing the file `test/test`. I suggest test creating the directoryseparately to avoid confusion for our users.